Podcast/TabbedPageViewController.swift

```diff
@@ -18,29 +19,25 @@ protocol TabbedPageViewControllerScrollDelegate: class {
 
 protocol TabbedViewControllerSearchResultsControllerDelegate: class {
     func didTapOnSeriesCell(series: Series)
-    func didTapOnTagCell(tag: Tag)
+    func didTapOnUserCell(user: User)
     func didTapOnEpisodeCell(episode: Episode)
 }
 
-protocol SearchRequestsDelegate: class {
-    func didRequestSearch(text: String)
-}
-
-class TabbedPageViewController: UIViewController, UIPageViewControllerDataSource, UIPageViewControllerDelegate, UISearchResultsUpdating, TabBarDelegate, SearchTableViewControllerDelegate, UINavigationControllerDelegate, UITableViewDelegate, UITableViewDataSource {
+class TabbedPageViewController: ViewController, UIPageViewControllerDataSource, UIPageViewControllerDelegate, UISearchResultsUpdating, TabBarDelegate, SearchTableViewControllerDelegate, UINavigationControllerDelegate {
     
-    let TabBarHeight: CGFloat = 44
+    let tabBarHeight: CGFloat = 44
+    let tabBarY: CGFloat = 75
     
     var viewControllers: [UIViewController]!
     
     weak var tabDelegate: TabbedPageViewControllerDelegate?
     weak var scrollDelegate: TabbedPageViewControllerScrollDelegate?
     weak var searchResultsDelegate: TabbedViewControllerSearchResultsControllerDelegate?
-    weak var searchRequestsDelegate: SearchRequestsDelegate?
     var tabBar: UnderlineTabBarView!
-    let tabSections: [SearchType] = [.episodes, .series, .people, .tags]
+    static let tabSections: [SearchType] = [.episodes, .series, .people]
```

Comment:
We still want the topics search though? You'd just only search among the set topics and subtopics we have.

Reply:
actually no! we decided to take searching on topics out. We can sync up tomorrow to go over that